DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/device_global/src/device_global.cpp

```diff
@@ -11,44 +11,41 @@
 
 #include "exception_handler.hpp"
 
-constexpr size_t kNumCounters = 4;
-constexpr int kInitialValue = 10;
-constexpr unsigned kNumIncrements = 3;
+constexpr size_t kNumIterations = 4;
+constexpr unsigned kNumWeightIncrements = 3;
+constexpr unsigned kVectorSize = 4;
 
 namespace exp = sycl::ext::oneapi::experimental;
 
-using IntScalar = std::array<int, 1>;
+using IntScalar = std::array<int, kVectorSize>;
 using FPGAProperties =
```

Comment:
```suggestion
using WeightsDeviceGlobalProperties=
```

choose a more informative name